src/PolyfillTrait6.php

```diff
@@ -18,7 +18,7 @@ trait PolyfillTrait6
 {
     public function expectException($exception)
     {
-        if (\is_callable(['parent', 'expectException'])) {
+        if (\is_callable([parent::class, 'expectException'])) {
```

Comment:
can we have a PHP CS Fixer rule that is automating that, pls? :)

Reply:
There is no way to determine whether `['parent', 'expectException']` is supposed to be a callable or just an array.

Reply:
but if it's passed as literal to `is_callable` etc, we can do the fix